server/src/__tests__/donatedItem.test.ts

```diff
@@ -16,38 +16,48 @@ describe('DonatedItem API Tests', () => {
         // Setup mock responses
         mockPrismaClient.program.findUnique.mockResolvedValue(newProgram);
         mockPrismaClient.donor.findUnique.mockResolvedValue(newDonor);
-        mockPrismaClient.donatedItem.create.mockResolvedValue({ id: 1, ...newItem });
-        mockPrismaClient.donatedItem.update.mockResolvedValue({ id: 1, ...updateItem });
+        mockPrismaClient.donatedItem.create.mockResolvedValue({
+            id: 1,
+            ...newItem,
+        });
+        mockPrismaClient.donatedItem.update.mockResolvedValue({
+            id: 1,
+            ...updateItem,
+        });
     });
 
     // Test POST /donatedItem
     it('validates Program name and Donor email before creating a donated item', async () => {
         const response = await request(app).post('/donatedItem').send(newItem);
         expect(response.status).toBe(201);
         expect(mockPrismaClient.donatedItem.create).toHaveBeenCalled();
-    
     });
 
     it('handles errors when the provided Program or Donor does not exist', async () => {
-        const programId = 99; 
+        const programId = 99;
         const donorId = 29;
         mockPrismaClient.program.findUnique.mockResolvedValue(null);
         mockPrismaClient.donor.findUnique.mockResolvedValue(null);
-    
-        const response = await request(app).post('/donatedItem').send({
-            ...newItem,
-            programId,
-            donorId,
-            dateDonated: new Date().toISOString()
-        });
-        expect(response.status).toBe(400);
-        expect(response.body.error).toContain(`Donor with ID: ${donorId} does not exist.`);
 
+        const response = await request(app)
+            .post('/donatedItem')
+            .send({
+                ...newItem,
+                programId,
+                donorId,
+                dateDonated: new Date().toISOString(),
+            });
+        expect(response.status).toBe(400);
+        expect(response.body.error).toContain(
+            `Donor with ID: ${donorId} does not exist.`,
+        );
     });
 
     // Test PUT /donatedItem/details/{id}
     it('updates donated item details correctly', async () => {
-        const response = await request(app).put('/donatedItem/details/1').send(updateItem);
+        const response = await request(app)
+            .put('/donatedItem/details/1')
```

Comment:
Ensure that all fields updated in the PUT request are properly verified in the test.